backend/app/services/k_batch.py

```
@@ -145,53 +146,95 @@ def delete_resources(
+    """
     try:
-        if not os.path.exists(file_path):
-            logger.info(f"Resource file not found: {file_path}")
-            return None, None
+        with open(file_path, "r", encoding="utf-8") as f:
+            documents = list(yaml.safe_load_all(f))
+            # Filter out None/empty documents
+            documents = [doc for doc in documents if doc]
+            logger.info(f"Loaded {len(documents)} documents from {file_path}")
+            return documents
+    except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)